src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -22,8 +22,26 @@
  */
 @CommandLineProgramProperties(
         usage = "Takes an input BAM and reference sequence and runs one or more Picard " +
-                "metrics modules at the same time to cut down on I/O. Currently all programs are run with " +
-                "default options and fixed output extesions, but this may become more flexible in future.",
+                "metrics modules at the same time to cut down on I/O.  These include CollectAlignmentSummaryMetrics, " +
+                "CollectInsertSizeMetrics, QualityScoreDistribution,  MeanQualityByCycle, and " +
+                "CollectBaseDistributionByCycle.  Produces a pdf and a txt file for each tool with the exception of the " +
```

Comment:
Geraldine: How do you suggest I handle the documentation?  Shall I wait or combine the tools now?

Thanks,
-David